src/site/xdoc/server/config-jmap.xml

```diff
@@ -43,6 +43,14 @@
                     <dt><strong>enabled</strong></dt>
                     <dd>true/false. Governs whether JMAP should be enabled</dd>
 
+                    <dt><strong>jmap.port</strong></dt>
+                    <dd>Optional. Defaults to 80. The port this server will be listening on. This value must be a valid
+                    port, ranging between 1 and 65535 (inclusive)</dd>
+
+                    <dt><strong>jmap.wiretap</strong></dt>
```

Comment:
I don't like having logging configuration spread in several places.
We could define a specific logger for that (say "org.apache.james.jmap.wire") and enable `wiretap` when this logger is in DEBUG mode (you can query logger configuration in Java code easily).

WDYT?

Reply:
> (you can query logger configuration in Java code easily)

Reference?

> WDYT?

Good idea

> say "org.apache.james.jmap.wire"

Maybe not that one because it's too easy to enable it by default but we can keep `http.jmap`

Reply:
LoggerFactory.getLogger("foo").isDebugEnabled()

Reply:
> Maybe not that one because it's too easy to enable it by default

or enable it only in TRACE?